From 79ebd5b1755d0646816cde42dbac92698ae3eda0 Mon Sep 17 00:00:00 2001
From: Balaji T K <balajitk@ti.com>
Date: Fri, 27 Sep 2013 17:05:10 +0530
Subject: [PATCH 45/58] ARM: dts: am335x-evm[sdk]: switch mmc1 to 4-bit mode

Set bus-width to make SD card operate in 4 bit mode.

Signed-off-by: Balaji T K <balajitk@ti.com>
Signed-off-by: Benoit Cousson <bcousson@baylibre.com>
---
 arch/arm/boot/dts/am335x-evm.dts   | 1 +
 arch/arm/boot/dts/am335x-evmsk.dts | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/am335x-evm.dts b/arch/arm/boot/dts/am335x-evm.dts
index 23b0a3e..028ca09 100644
--- a/arch/arm/boot/dts/am335x-evm.dts
+++ b/arch/arm/boot/dts/am335x-evm.dts
@@ -521,4 +521,5 @@
 &mmc1 {
 	status = "okay";
 	vmmc-supply = <&vmmc_reg>;
+	bus-width = <4>;
 };
diff --git a/arch/arm/boot/dts/am335x-evmsk.dts b/arch/arm/boot/dts/am335x-evmsk.dts
index bc93895..563a2b1 100644
--- a/arch/arm/boot/dts/am335x-evmsk.dts
+++ b/arch/arm/boot/dts/am335x-evmsk.dts
@@ -423,4 +423,5 @@
 &mmc1 {
 	status = "okay";
 	vmmc-supply = <&vmmc_reg>;
+	bus-width = <4>;
 };
-- 
1.8.4.rc3

